package.json

```diff
@@ -15,6 +15,7 @@
   "dependencies": {
     "@nuxtjs/axios": "^5.13.6",
     "core-js": "^3.25.3",
+    "moment": "^2.29.4",
```

Comment:
Just FYI moment is deprecated
we should look for new packages like Luxon

Reply:
noted